src/browser/api/window.js

```diff
@@ -502,6 +502,28 @@ Window.create = function(id, opts) {
             }
         });
 
+        const emitToAppAndWin = (...types) => {
+            let isMainWindow = (uuid === name);
+            types.forEach(type => {
+                ofEvents.emit(route.window(type, uuid, name), { topic: 'window', type, uuid, name });
+                if (isMainWindow) {
+                    ofEvents.emit(route.application(`window-${type}`, uuid), { topic: 'application', type, uuid, name });
```

Comment:
I'd like to put together an "event traffic" internal wiki to reduce the mystery...